app/views/entities/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="min_nav">
+  <%= link_to "", group_entities_path(@group), class: "fa fa-chevron-left fa-lg" %>
+  <h3 class="mb-3">New Transaction</h3>
+</div>
+<%= form_with model: @entity, url: group_entities_path, method: :post, data: { turbo: false } do |form| %>
+  <div class="mb-3">
+    <%= form.label :name %>
+    <%= form.text_field :name, class: 'form-control', placeholder: 'Transaction name' %>
+  </div>
+  <div class="mb-3">
+    <%= form.label :amount %>
+    <%= form.text_field :amount, class: 'form-control', placeholder: 12, type: "number", step: ".01" %>
+  </div>
+
+  <div class="mb-3">
+    <%= form.submit 'Save', class: 'btn btn-primary' %>
+  </div>
+<% end %>
```

Comment:
- [ ]  The form is missing a field where users can select the categories to which the transaction should belong. Kindly add a field to the form that allows the user to choose at least one category (the user should be able to choose more than one category).

Reply:
Hi @Chrissiku, I understand that you want to create something that interests you even though it does not follow to the project's guidelines. But as a developer, you are asked to follow the requirements of the project.

For better comprehension, the relationship between categories and transactions is a many-to-many relationship, and that means that the transaction can belong to many categories.

As required in the project requirements, the user should be able to fill out a form with name (mandatory), amount (mandatory), and categories (mandatory at least one) to create a new transaction.